### ISSUE TITLE
Discussion of large roughness

### ISSUE THREAD

**Issue by andyfaff** [reflectivity/edu_outreach#32]
See comments in #31. It's worth the community having a discussion about recommended practice when the thickness of a slab is not much greater than its roughness.

- when roughness is large is microslicing required? Does Nevot Croce become invalid?
- what model parameters would you report?
- can any valid model be created?

I'll kick off with a [simulation](https://gist.github.com/andyfaff/96062c396a3af051b6d7526869eccfe6). When does Nevot Croce become invalid and microslicing required? The reflectivity signals from both approaches are identical (from a data analysis point of view).

@acaruana2009

**Comment by bmaranville** [comment]
Yes, a discussion might be useful.  We have not made the same recommendation as in the paper about using NC roughness - in some cases it makes a lot of sense to use the roughness even when it is nearly as big or bigger than the layer thickness.  The best example is a thin layer somewhere in your material stack, that the film creator is sure is there but doesn't know how rough it is.  The NC terms give the correct limits when the roughness >> layer thickness, in that the reflection term from that layer goes to zero.

**Comment by andyfaff** [comment]
https://github.com/reflectivity/edu_outreach/pull/31#issuecomment-671613780

**Comment by aglavic** [comment]
Proposal for a few clear-cut from my POV:

1. Thin layers with comparable roughness on top of each other. Example would be thick rough buffer layer with high quality epitxy on top. E.g. Ag/Fe. Here the standard roughness would be physical and correct.
2. Thin rough layer on top of smooth layer. This should be some island-kind of structure with pretty normal SLD in dop direction and some sharp cut-off at the bottom interface. Could probably be described with some sort of slap model.
3. Non-gaussian height distribution like island growth. This would require some knowledge about how it grows and could than be slap-model like.
